src/common/enums.js

```diff
@@ -141,22 +141,26 @@ export const fileTypeDetail = {
   SHM: {
     icon: 'shipping',
     title: 'Shipping Manifest',
```

Comment:
Should this be called Sample Manifest? I might be wrong about this, but it doesn't seem like we get shipping manifests that much. We get sample manifests which link specimens to their participants and usually include sample attributes like anatomical site and composition. Do we even need this separate file type?

Reply:
I think the aim of this is that we _eventually_ want to receive and parse the Shipping Manifest as a sort of _baseline_ for accounting. 

> One idea I had around this was to make a manifest builder tool that exports to the shipping manifest format. This way we already have their specimen info early on